UmpleToJava/UmpleTLTemplates/JavaClassGenerator.ump

```diff
@@ -98,17 +98,27 @@ public <<# if (uClass.getIsAbstract()) { append(realSb, "{0} ", "abstract"); } #
 for (StateMachine smq : uClass.getStateMachines())
   {
     if (smq.isQueued())
-    {
-      append(realSb," implements Runnable");
+    { 
+      if (uClass.hasParentInterface() == false){
```

Comment:
I wonder if you can merge it or not?
If not, I checked your PR, It doesn't show that wrong behavior and I can merge it.